[(Error (py-list 1 2) Exception caught:
TypeError: py_list() takes 1 positional argument but 2 were given

At:
  /home/me/.local/lib/python3.10/site-packages/hyperon/atoms.py(428): execute
  /home/me/.local/lib/python3.10/site-packages/hyperon/atoms.py(203): _priv_call_execute_on_grounded_atom
  /home/me/.local/lib/python3.10/site-packages/hyperon/runner.py(207): run
  /home/me/.local/lib/python3.10/site-packages/hyperon/metta.py(35): main
  /home/me/.local/bin/metta(8): <module>
)]
